### ISSUE TITLE
predict 每一次结果都不一样

### ISSUE THREAD

**Issue by lun-lun-byte** [PaddlePaddle/PaddleNLP#1671]
Hello PaddlePaddle2.0.1，百度自己的 aistudio

在实现文本多标签分类时（https://aistudio.baidu.com/aistudio/projectdetail/2540472?channelType=0&channel=0），训练过程还挺好的，也可以正常的 loss 下降，并保存模型，但是 predict 就很奇怪，每一次结果都不一样。具体代码如下：

![image](https://user-images.githubusercontent.com/78267675/153741999-177d8f44-3788-46aa-a8e6-ccd97ddb2561.png)

from paddlenlp.transformers import ErnieTokenizer, ErnieModel
# 本实验使用的是ERNIE模型
model_name_or_path="ernie-1.0"
pretrained_model = ppnlp.transformers.ErnieModel.from_pretrained(model_name_or_path)
tokenizer = ppnlp.transformers.ErnieTokenizer.from_pretrained(model_name_or_path)

model = MultiLabelClassifier(pretrained_model, num_labels=1262)
state_dict = paddle.load("checkpoint/epoch_12.pdparams")
model.set_dict(state_dict)
model.eval()
print("加载成功了模型呀！")
probs, preds = predict(model, test_data_loader, label_info, batch_size=1)

每一次结果都不一样，这是为什么呢？

**Comment by ZeyuChen** [comment]
是否可以尝试下升级到最新的paddle版本2.1以上和最新的paddlenlp 2.2版本后再对比试试？ @lun-lun-byte
